### ISSUE TITLE
Preselect closing element tag

### ISSUE THREAD

**Issue by KirillOsenkov** [mhutch/MonoDevelop.MSBuildEditor#149]
There's some friction around adding end element tags:

![image](https://github.com/mhutch/MonoDevelop.MSBuildEditor/assets/679326/7a573389-c0af-48b2-9577-a1fe4fdac489)

would be nice to preselect `</Target>` here

**Comment by KirillOsenkov** [comment]
It works great if you overtype the closing `>` in `<Target>`, but if you just move the caret to the right of it using the right arrow, it doesn't autocomplete `</Target>` and there's some ceremony in typing it in properly

**Comment by mhutch** [comment]
The problem of the comment item being committed but the `/` should be fixed by #153. Was there any other reason you wanted to preselect the closing tag?

**Comment by KirillOsenkov** [comment]
yes because when the tag is unclosed I always want to close it first by typing </Foo>

see how this works in various editors in VS, for xml, xaml or html

**Comment by KirillOsenkov** [comment]
sometimes pressing Enter after an unclosed open tag inserts the closing tag automatically, perhaps this should happen here too
